lib/OpenQA/WebAPI/Controller/Running.pm

```diff
@@ -118,14 +112,14 @@ sub streamtext {
     # Check for new lines from the logfile using recurring timer
     # Setup utility function to close the connection if something goes wrong
     my $timer_id;
-    my $close_connection = sub {
+    my $close_connection = sub ($self) {
```

Comment:
What do you guys think: Is this function just not properly collected in coverage or is it unused code?

Reply:
excluding "Running" from this PR for now.

Reply:
-> https://github.com/os-autoinst/openQA/pull/4898